gradle/common-java.gradle

```diff
@@ -40,8 +40,8 @@ if (java6JreDir) {
         bootClasspath += "$archivePath;"
     }
     tasks.withType(JavaCompile) {
-        sourceCompatibility = 1.6
-        targetCompatibility = 1.6
+        sourceCompatibility = 1.8
```

Comment:
So you dropped 1.7 support?

Reply:
I made it so because sometimes in some situations it the 1.7 target code would not produce the desired exceptions because these changes were mandated from JDK 1.8. Also, we need to decide till what point do we need to support backward compatibility in the SDK. We might not be able to use all the new features of JDK 8 like automatic inference of types in <> operator, passing of functions using lambda expressions etc which is modern java.

Reply:
If required no harm in reverting it back if things work fine. However at one point we need to make this decision.